hangman.py

```diff
@@ -0,0 +1,217 @@
+
+    * The user should start with 6 guesses
+
+    * Before each round, you should display to the user how many guesses
+      s/he has left and the letters that the user has not yet guessed.
+
+    * Ask the user to supply one guess per round. Remember to make
+      sure that the user puts in a letter!
+
+    * The user should receive feedback immediately after each guess
+      about whether their guess appears in the computer's word.
+
+    * After each guess, you should display to the user the
+      partially guessed word so far.
+
+    Follows the other limitations detailed in the problem write-up.
+    '''
+
+    guesses_remaining = 6
+    warnings_remaining = 3
+    letters_guessed = []
+    print(f'''Welcome to the game Hangman!
+I am thinking of a word that is {len(secret_word)} letters long.
+You have {warnings_remaining} warnings left''')
+    while (guesses_remaining > 0) and not (is_word_guessed(secret_word, letters_guessed)):
+        print(f'''-------------
+You have {guesses_remaining} guesses left.
+Available letters: {get_available_letters(letters_guessed)}''')
+
+        letters_guessed.append((input("Please guess a letter:")).lower())
+
+        #Cheking for symbols quantity
+        if len(letters_guessed[-1])>1:
+            print("Your input has more then one symbol")
+
+        #Add hints, output all the aceptable words
+        if letters_guessed[-1]=="*":
+            show_possible_matches(get_guessed_word(secret_word, letters_guessed))
+
+        #Checking for unwanted symbols
+        elif not letters_guessed[-1].isalpha():
+            if warnings_remaining == 0:
+                guesses_remaining -= 1
+            else:
+                warnings_remaining -= 1
+            print("Oops! That is not a valid letter. You have", warnings_remaining, "warnings left:", end="")
+
+        #Cheking for repeating symbols
+        elif letters_guessed[-1] in letters_guessed[0:-1]:
+            if warnings_remaining == 0:
+                guesses_remaining -= 1
+            else:
+                warnings_remaining -= 1
+            print("Oops! You've already guessed that letter. You have", warnings_remaining, "warnings left:", end="")
+
+        #Cheking for usuall mistake
+        elif not (letters_guessed[-1] in secret_word):
+            if letters_guessed[-1] in "aeiou":
+                guesses_remaining -= 2
+            else:
+                guesses_remaining -= 1
+            print("Oops! That letter is not in my word:", end="")
+
+        #Cheking for right answer
+        elif letters_guessed[-1] in secret_word:
+            print("Good guess:", end="")
+
+        print(get_guessed_word(secret_word, letters_guessed))
+
+    if is_word_guessed(secret_word, letters_guessed):
+        print("Congratulations, you won! Your total score for this game is:", guesses_remaining * len(secret_word))
+    else:
+        print("Sorry, you ran out of guesses. The word was else\nIt's", secret_word)
```

Comment:
Could you have several prints here? It would be more readable

Reply:
+